### PR TITLE
fix(test/*): fixed tests on windows

### PR DESCRIPTION
**Pull request by GitScrum**

### CODE REVIEW

**Comment by SamVerschueren** [comment]
So there isn't something like executable in windows?

File: test/cli.js [hunk 1]
```diff
@@ -12,14 +12,16 @@ test('no error', async t => {
 });
 
 test('file is not executable', async t => {
-	t.deepEqual(await m('not-executable', opts), [
+	const fixtures = process.platform === 'win32' ? [] : [
```
Comment (SamVerschueren) [inline test/cli.js]: rename to `expectedResult`

File: test/test.js [hunk 2]
```diff
@@ -42,6 +42,6 @@ test('unknown plugin', t => {
 	t.throws(m('.', {cwd: '../', plugins: ['foo'], rules: {foo: 'error'}}), 'Could not find module for plugin \'foo\'.');
 });
 
-test('cli', async t => {
-	t.throws(execa('../cli.js', ['fixtures/package/no-files', '--no-inherit']), /[ ]*?✖[ ]*?Missing files property in package.json.[ ]*pkg-files/);
+test('cli', t => {
+	t.throws(execa('../cli.js', ['fixtures/package/no-files', '--no-inherit']), /[ ]*?✖|×[ ]*?Missing files property in package.json.[ ]*pkg-files/);
```
Comment (SamVerschueren) [inline test/test.js]: Use [figures](https://github.com/sindresorhus/figures) for in combination with `new RegExp()`. Seems more elegant.

Comment (GitScrum) [inline test/test.js]: Thanks, i change.

**Comment by GitScrum** [comment]
> So there isn't something like executable in windows?

Windows doesn't look at chmod)) he probably looks in the registry. In this case, he believes it is executable.

**Comment by GitScrum** [comment]
@SamVerschueren may be use [appveyor](http://www.appveyor.com/) for testing on windows ?

**Comment by SamVerschueren** [comment]
> may be use appveyor for testing on windows ?

Adding it now. Thanks for the PR! 🍻

**Pull request merged**